### ISSUE TITLE
DreamBooth: Fine Tuning Text-to-Image Diffusion Models for Subject-Driven Generation

### ISSUE THREAD

**Issue by nmhkahn** [nmhkahn/paper-reading#43]
## Motivation
- 몇장의 입력 이미지를 가지고 pre-trained diffusion 모델 fine-tune -> personalized model

## Method
<img width="650" alt="Screenshot 2022-12-21 at 3 50 56 PM" src="https://user-images.githubusercontent.com/16869368/208839939-e04edc2c-e9c9-4253-9b02-691ba98cd865.png">

- 스페셜 토큰 [V]와 입력 이미지를 가지고 T2I 모델 fine-tune
- Textual inversion이랑 다르게 모델 자체도 fine-tune하는 점이 특징
- 학습을 용이하게 하기 위해 스페셜 토큰 [V]는 rare하게 등장하는 단어를 사용하는게 좋다고 함
- Prior preservation loss: 그냥 모델을 fine-tune하면 overfit, language drift 등의 문제 발생 우려 -> e.g. 모든 강아지 생성 결과가 내가 입력한 사진의 강아지 모습으로 나옴
    - Pre-trained T2I가 생성한 결과도 fine-tune된 T2I가 생성할 수 있어야함
    - 유저가 입력한 이미지에 해당되는 클래스를 프롬프트로 주고 pre-trained과 fine-tuned 모델에게 이미지 생성하게 한뒤, 두 결과에 loss를 constraint으로 걸어줌

**Comment by nmhkahn** [comment]
## Discussion
- 정확하지는 않겠지만 dreambooth로 fine-tune 하면 pivotal하게 학습이 되는 느낌을 받음
- 예를들어, prior preservation loss도 어느 특정 강아지 분포만 내가 입력으로 준 강아지처럼 나오게 하는 등등
- 그러면 하나의 모델에 여러 pivotal tuning을 적용해서 한 모델이 여러 personalized 모델처럼 행동하게 할 수 있지 않을까?
- MyStyle: A Personalized Generative Prior 논문 참고하기
